modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -165,7 +176,13 @@ struct GAPI_EXPORTS_W_SIMPLE GCompileArg
         return util::any_cast<T>(arg);
     }
 
+    void serialize(cv::gapi::s11n::IOStream& os) const
+    {
+        serializeF(os, *this);
+    }
+
 private:
+    std::function<void(cv::gapi::s11n::IOStream&, const GCompileArg&)> serializeF;
```

Comment:
1. problem is here
2. and in `GAPI_EXPORTS_W_SIMPLE` in GCompileArg

Reply:
Can you please explain what the problem is here? (CC: @TolyaTalamanov )